src/compiler/sys.ts

```diff
@@ -282,11 +282,16 @@ namespace ts {
                     };
 
                     function fileChanged(curr: any, prev: any) {
+                        // mtime.getTime() equals 0 if file was removed
+                        if (curr.mtime.getTime() === 0) {
+                            callback(fileName, true);

```

Comment:
Add comment with parameter name in front of boolean arguments (e.g. `callback(fileName, /* removed */ true);`